app/views/comments/_form.html.erb

```diff
@@ -2,17 +2,17 @@
 <%= render partial: "shared/form_errors", object: @comment %>
 <%= form_for [ @article, @comment ] do |f| %>
 
-  <p>
+  <div>
     <%= f.label :author_name %><br/>
-    <%= f.text_field :author_name %>
-  </p>
+    <%= f.text_field :author_name, required: :true, placeholder: :"Add your name here" %>
```

Comment:
For the placeholder text, just a string is fine:

`placeholder: "Add your name here"`